src/cpp/core/Database.cpp

```diff
@@ -1039,6 +1040,10 @@ Error SchemaUpdater::update()
 
       SchemaVersion currentVersion;
       error = databaseSchemaVersion(&currentVersion);
+
+      LOG_DEBUG_MESSAGE("Database Schema Version:\t\t" + currentVersion.Date + " : " + currentVersion.Flower);
+      LOG_DEBUG_MESSAGE("Highest Migration Schema Version:\t" + migrationVersion.Date + " : " + migrationVersion.Flower);
```

Comment:
Should we log this as all one message? Or maybe even just some minor wording changes
```suggestion
      LOG_DEBUG_MESSAGE("Current Database Schema Version:\t\t" + currentVersion.Date + " : " + currentVersion.Flower);
      LOG_DEBUG_MESSAGE("Highest Available Database Schema Version:\t" + migrationVersion.Date + " : " + migrationVersion.Flower);
```